### PR TITLE
o [NXCM-4120] Make user profile tabs bookmarkable

### PR DESCRIPTION
**Pull request by nabcos**

### CODE REVIEW

**Comment by adreghiciu** [comment]
+0 (I have no clue what the JS does)

**Comment by peterlynch** [comment]
+1

File: nexus/nexus-webapp/src/main/webapp/js/nexus/error/ErrorHandler.js [hunk 1]
```diff
@@ -56,7 +56,7 @@ Nexus.error.ErrorHandler = function() {
         args[x] = arguments[x];
       }
       try {
-        if ( '?debug' === windows.location.search )  {
+        if ( '?debug' === window.location.search )  {
```
Comment (peterlynch) [inline nexus/nexus-webapp/src/main/webapp/js/nexus/error/ErrorHandler.js:59]: what does this affect?

**Comment by peterlynch** [comment]
so will #profile without additional ";$viewName" go to Summary panel by default? Otherwise how will NXCM-3842 be fixed?

File: nexus/nexus-webapp/src/main/webapp/js/nexus/error/ErrorHandler.js (hunk 1 above)
Comment (nabcos) [inline nexus/nexus-webapp/src/main/webapp/js/nexus/error/ErrorHandler.js:59]: Whoops, would have been best in its own branch. That's the js error dialog popup you can get with `index.html?debug`, enabling it did not work.

**Comment by nabcos** [comment]
The first registered view will be selected, and that's the Summary view.

**Pull request merged**